sdk/textanalytics/Azure.AI.TextAnalytics/tests/TextAnalyticsClientLiveTests.cs

```diff
@@ -560,6 +564,7 @@ public async Task RecognizeEntitiesTest()
         }
 
         [Test]
+        [Ignore ("Tracked by issue: https://github.com/Azure/azure-sdk-for-net/issues/11567")]
```

Comment:
You can pass in model version and get the same response as before. Python did this, since it's more of a service side bug and not sdk side: 
https://github.com/Azure/azure-sdk-for-python/blob/9da31763f9809305c4e04ac41b7a6a995dcdeb9e/sdk/textanalytics/azure-ai-textanalytics/tests/test_recognize_entities.py#L39

Reply:
Thanks Isa! Will do in a next pass of PRs